ide/project.ant.ui/src/org/netbeans/spi/project/support/ant/ui/LicenseHeadersPanel.java

```diff
@@ -22,6 +22,7 @@
 import java.awt.event.ActionListener;
 import java.io.IOException;
 import javax.swing.Action;
+import javax.swing.ListModel;
```

Comment:
Looks unnecessary.

Reply:
good eyes.. I sometimes miss things like this since I'm editing so many files at once..

thanks!